Hunting Queries/AzureDiagnostics/CriticalPortsOpened.yaml

```diff
@@ -0,0 +1,38 @@
+id: 66d494c0-233c-438a-9b1b-5fe839790d38
+name: Check critical ports opened to the entire internet
+description: |
+  'Discover all critical ports from a list having rules like 'Any' for sourceIp, which means that they are opened to everyone. Critial ports shouldn't be opened to everyone, and should be filtered.'
+requiredDataConnectors:
+  - connectorId: AzureDiagnostic
+tactics:
+  - InitialAccess
+query: |
+
+  //Check critical ports opened to the entire internet
+  AzureDiagnostics
+  | where Category == "NetworkSecurityGroupEvent" 
```

Comment:
There are many hits for "blocks" if not excluded.  I would recommend that you include the following, if you want to leave blocks in, I would include it, comment it out and then put a note above it about when and how to use it.

| where type_s !~ "block"

Reply:
done